ProcessMaker/processmaker
# Fix Sorting Error in My Templates and Public Templates

This PR addresses errors that occur when users attempt to sort the templates by columns in both the My Templates and Public Templates sections. The issue stemmed from a missing method, `handleEllipsisClick`, which was required to handle sorting options when clicking on table column headers.

## Solution
- Implemented the `handleEllipsisClick` method to handle sorting options when selecting table column headers.

## How to Test

1. Navigate to both the My Templates and Public Templates sections.
2. Sort the templates by columns by clicking on the column headers.
3. Verify that no errors occur during the sorting process.
4. Ensure that the templates are sorted correctly based on the selected column.

## Related Tickets & Packages
- [FOUR-14645](https://processmaker.atlassian.net/browse/FOUR-14645)

ci:next
ci:deploy

## Code Review Checklist


[FOUR-14645]: https://processmaker.atlassian.net/browse/FOUR-14645?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
